operator/src/main/resources/scripts/model_wdt_mii_filter.py

```diff
@@ -385,6 +385,11 @@ def customizeServerIstioNetworkAccessPoint(server, listen_address):
   if istio_readiness_port is None:
     return
   admin_server_port = server['ListenPort']
+  # Set the default if it is not provided to avoid nap default to 0 which fails validation.
+
+  if admin_server_port is None:
+    admin_server_port = 7001
```

Comment:
What happens if the domain is in SecuredMode?

Reply:
istio won't work with secure mode yet

Reply:
Oh! That seems like an issue. Do we have a JIRA to complete that support? I'd like to resolve that gap before the Verrazzano team hits it.

Reply:
@ddsharpe @jshum2479, what would be the correct behavior for secure mode? Does this "None" check let us differentiate between when the customer has left it blank or if the admin server's default port is disabled because of secure mode?

Reply:
Note:  the filter code already have provision to setup the ssl port  if the secure mode is enabled (line 423).    

But, ultimately it will not work in istio environment. 

1) whenever admin port (same for secure mode) is enabled, the readiness probe /weblogic/ready is treated as management function because it started with /weblogic and it must be accessed directly read-address:adminport and not proxied it through localhost:adminport
2) Istio always proxy it through unless there is annotation to forbid the rewrite the port traffic (essentially take it out of the mesh).
3) operator current implementation always use the plain readiness port in the domain spec,  while it can be fixed in PodStepContext but it won't fix (2).

Reply:
As for the correct behavior,  I am not sure what's the correct behavior.  If secure mode is enabled, does it mean regular listen port is disabled?

Reply:
Yes, that's right: https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.3/lockd/secure.html#GUID-F980DB67-7CDE-4EF8-986D-D188D4EDB706

Reply:
So the core should automatically disabled the listenPort.  The question is whether the operator needs to do anything special about it.   I suggest creating another issue to handle secure mode as I suspect there maybe issue in non-istio case.  This PR is for missing listenPort only.  For adminport/secure mode it's a dead end for istio for now.